pkg/apis/admission/types.go

```diff
@@ -92,6 +92,12 @@ type AdmissionResponse struct {
 	// PatchType indicates the form the Patch will take. Currently we only support "JSONPatch".
 	// +optional
 	PatchType *PatchType
+	// Annotations is an unstructured key value map set by remote admission controller (e.g. error=image-blacklisted).
+	// MutatingAdmissionWebhook and ValidatingAdmissionWebhook admission controller will fill in the Annotations with
+	// admission webhook name (e.g. imagepolicy.example.com/error=image-blacklisted). Annotations may be provided by
+	// the admission webhook to add additional context to the audit log for this request.
+	// +optional
+	Annotations map[string]string
```

Comment:
will this be confused with the Annotations field on the API object being admitted? If we leave this named "Annotations", the documentation needs to be very clear that this is only audit logging. Alternately we could name it something like `AuditAnnotations`.

Reply:
We should also clarify the documentation on the audit.Event type that the map does not contain the annotations of the submitted object

Reply:
> AuditAnnotations is ok too.

Done.

> We should also (in a separate PR) clarify the documentation on the audit.Event type that the map does not contain the annotations of the submitted object

Will do this after https://github.com/kubernetes/kubernetes/pull/65891 gets merged.

Reply:
#65891 is merged, can we queue this up?

Reply:
Done.
https://github.com/kubernetes/kubernetes/pull/67386